tools/testers/algorithm-tester.pl

```diff
@@ -244,6 +234,19 @@ sub run_test {
         for my $x (@{$t->{tests}}) {
             process_single_test($x, $dir,, $DBNAME, \%res)
         }
+        if ($OS =~ /msys/ || $OS=~/MSW/) {
+            for my $x (@{$t->{windows}}) {
+                process_single_test($x, $dir,, $DBNAME, \%res)
+            }
+        } elsif ($OS=~/Mac/) {
```

Comment:
you might want to check if this is correct by 
comparing what you get on the print.